docs/security.rst

```diff
@@ -103,7 +103,7 @@ way to start a HTTPS server when using Connexion?
 .. _rfc7662: https://tools.ietf.org/html/rfc7662
 .. _minimal OAuth example application: https://github.com/zalando/connexion/tree/master/examples/swagger2/oauth2
 .. _minimal Basic Auth example application: https://github.com/zalando/connexion/tree/master/examples/swagger2/basicauth
-.. _minimal API Key example application: https://github.com/zalando/connexion/tree/master/examples/oauth2/apikey
+.. _minimal API Key example application: https://github.com/zalando/connexion/blob/master/examples/openapi3/apikey/app.py
```

Comment:
This should point to the directory, not the python file.

Reply:
Fixed.